include/ts/ts.h

```diff
@@ -2601,6 +2602,12 @@ tsapi TSReturnCode TSRemapFromUrlGet(TSHttpTxn txnp, TSMLoc *urlLocp);
 //
 tsapi TSReturnCode TSRemapToUrlGet(TSHttpTxn txnp, TSMLoc *urlLocp);
 
+// Override response behavior, and hard-set the state machine for whether to succeed or fail, and how.
+tsapi void TSHttpTxnResponseActionSet(TSHttpTxn txnp, TSResponseAction action);
```

Comment:
Pass by value? `TSResponseAction const* action`.

Reply:
I was on the fence. I went with value, because it needs copied anyway. But yeah, it's going to be 2 copies, that the compiler probably can't optimize. I don't object to changing it

Reply:
Just from an API consistency point of view, it should be a pointer.

Reply:
Done